exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the input map from a map of value: [ keys ] to a map of key: value for each of the keys",
```

Comment:
The way the function is invoked is very language specific and will need to be specified by the language anyway.

@petertseng makes some good points above and I am sympathetic to the idea that some kind of identifier for the name of the problem (or section of the problem) can be specified in the JSON.

> until we come up with a proposed unified schema for all the JSON files.

 :+1:

So I guess what you're doing here is fine for now.